### PR TITLE
Phillyjug blog

### PR DESCRIPTION
**Pull request by ajm01**

### CODE REVIEW

File: posts/2021-10-05-openliberty-jug-presentation.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: post
-title: "Why Open Liberty is Your Next Cloud Native Java Runtime"
+title: "A Fun Cup of Joe: Find Out Why Open Liberty is Your Next Cloud Native Java Runtime"
 # Do NOT change the categories section
 categories: blog
-author_picture: 
+author_picture: https://avatars3.githubusercontent.com/ajm01
 author_github: https://github.com/ajm01
 seo-title: Why Open Liberty is Your Next Cloud Native Java Runtime - OpenLiberty.io
```
Comment (gljrobinson) [inline posts/2021-10-05-openliberty-jug-presentation.adoc]: This should match the title above

Comment (gljrobinson) [reply posts/2021-10-05-openliberty-jug-presentation.adoc]: Same for the seo-description below

File: posts/2021-10-05-openliberty-jug-presentation.adoc [hunk 2]
```diff
@@ -14,18 +14,21 @@ additional_authors:
...
 
-The focus of the presentation centered around Open Liberty as an open framework for Java developers that highly optimizes the development of microservices using Eclipse MicroProfile and Jakarta EE APIs. To highlight Open Liberty as a flexible, fast and lightweight Java runtime, the discussion touched on important Open Liberty framework design attributes such as its simple configuration to enable "Just Enough Runtime" - configuring the framework to enable only the subset of features an application needs at runtime, its low operating cost due to a reduced memory footprint based on that minimal configuration, zero migration, Kubernetes optimizations and its efficient rapid developer experience. Additionally, the fact that Open Liberty supports the deployment of Spring Boot applications generated an enthusiastic question from a member of the audience interested in understanding how Open Liberty could help a shop with hundreds of Spring Boot applications become more efficient.
+image::img/blog/cupofjoe.JPG[A Fun Cup Of Joe, align="center"]
```
Comment (gljrobinson) [inline posts/2021-10-05-openliberty-jug-presentation.adoc]: Not entirely sure, but I think images are supposed to be in png format but I'll let David comment on that when he does his editorial review.

**Review by gljrobinson — COMMENTED** [review]
Few comments to take a look at but it's looking great. Thanks for your work on this

File: posts/2021-10-05-openliberty-jug-presentation.adoc (hunk 1 above)
Comment (ajm01) [reply posts/2021-10-05-openliberty-jug-presentation.adoc]: should all be in sync now

File: posts/2021-10-05-openliberty-jug-presentation.adoc (hunk 2 above)
Comment (ajm01) [reply posts/2021-10-05-openliberty-jug-presentation.adoc]: i just changed it to a png

**Review by gljrobinson — APPROVED** [review]
LGTM

**Pull request merged**